### DISCUSSION TITLE
export some info from MS Excel file

### DISCUSSION THREAD

**Discussion by jungle-boogie** [saulpw/visidata#1352] [Q&A]
Hi,

I have MS Excel files with some data that I would like to extract and save to a text file using the replay and/or batch mode.

There is more data available in the sheet than I want, but below is what I want to export out of the sheet:

```
foo | 300
bar | 200
baz | 1000
```

I can't seem to create a new sheet and paste the data in because of this issue:
https://github.com/saulpw/visidata/issues/1348

Is there an alternative to copy/paste this info, so that I can save it as a csv file?

Thanks!

**Comment by saulpw** [comment]
Hi @jungle-boogie, you can use `g'` to freeze the whole sheet first, so that it's not an xlsx-specific sheet anymore, and then copy/paste as any other sheet.
